tests/backup/backup_rbac_test.go

```diff
@@ -127,6 +153,105 @@ var _ = Describe("{VerifyRBACForInfraAdmin}", func() {
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Suspending Backup Schedule [%s] for user [%s]", backupScheduleWithLabel, infraAdminUser))
+			infraAdminLabelledBackupNames = append(infraAdminLabelledBackupNames, scheduledBackupNameWithLabel)
+		})
+
+		Step("Validate restoring the scheduled backup with namespace label", func() {
+			log.InfoD("Validate restoring the scheduled backup with namespace label")
+			nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+			log.FailOnError(err, "failed to fetch user %s ctx", infraAdminUser)
+			multipleBackupNamespace, err := FetchNamespacesFromBackup(nonAdminCtx, scheduledBackupNameWithLabel, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching namespaces %v from backup %v", multipleBackupNamespace, scheduledBackupNameWithLabel))
+			multipleRestoreMapping = make(map[string]string)
+			for _, namespace := range multipleBackupNamespace {
+				restoredNameSpace := fmt.Sprintf("%s-%v", scheduledBackupNameWithLabel, RandomString(3))
+				multipleRestoreMapping[namespace] = restoredNameSpace
+			}
+			customRestoreName = fmt.Sprintf("%s-%v", "customrestore", RandomString(4))
+			err = CreateRestore(customRestoreName, scheduledBackupNameWithLabel, multipleRestoreMapping, destinationClusterName, orgID, nonAdminCtx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying multiple backup restore [%s] in custom namespace [%v]", customRestoreName, multipleRestoreMapping))
+			infraAdminLabelledRestoreNames = append(infraAdminLabelledRestoreNames, customRestoreName)
+		})
+
+		Step(fmt.Sprintf("Validate deleting namespace labelled backups and restores from the infra admin %s user context", infraAdminUser), func() {
+			log.InfoD(fmt.Sprintf("Validate deleting namespace labelled backups and restores from the infra admin user %s user context", infraAdminUser))
+			nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+			log.FailOnError(err, "failed to fetch user %s ctx", infraAdminUser)
+			for _, backupName := range infraAdminLabelledBackupNames {
+				backupUid, err := Inst().Backup.GetBackupUID(nonAdminCtx, backupName, orgID)
+				log.FailOnError(err, "Failed to fetch the backup %s uid of the user %s", backupName, infraAdminUser)
+				_, err = DeleteBackup(backupName, backupUid, orgID, nonAdminCtx)
+				log.FailOnError(err, "Failed to delete the backup %s of the user %s", backupName, infraAdminUser)
+				err = DeleteBackupAndWait(backupName, nonAdminCtx)
+				log.FailOnError(err, fmt.Sprintf("waiting for backup [%s] deletion", backupName))
+			}
+			for _, restoreName := range infraAdminLabelledRestoreNames {
+				err := DeleteRestore(restoreName, orgID, nonAdminCtx)
+				dash.VerifySafely(err, nil, fmt.Sprintf("Verifying the deletion of the restore named [%s]", restoreName))
+			}
+		})
+
+		Step(fmt.Sprintf("Delete Infra Admin %s backup schedule with label ", infraAdminUser), func() {
```

Comment:
can we update checks and steps to cleanup backup Schedules tearDown() so that other cases don't fail ?